hawk/src/main/java/com/orhanobut/hawk/HawkBuilder.java

```diff
@@ -136,6 +139,10 @@ public boolean isEncrypted() {
     return encryptionMethod != EncryptionMethod.NO_ENCRYPTION;
   }
 
+  Status getStatus() {
```

Comment:
Hawk.java has already isInitialised method, what is the difference? I think they do the same job, we can make this one private, and since it's package-private, it is not access-able already for publicly.